force-app/main/default/objects/Course_Registration__c/validationRules/Do_not_exceed_max_number_of_participants.validationRule-meta.xml

```diff
@@ -0,0 +1,20 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<ValidationRule xmlns="http://soap.sforce.com/2006/04/metadata">
```

Comment:
Hvordan fungerer dette med nav.no og påmeldinger? Vil det stå i listen på nav.no at det er fult, slik at man ikke kan søke? Hvis ikke, så vil brukeren få beskjed etter alt er fylt ut, at det er fullt. Det er jo litt dumt. Vil du sjekke om det?

Reply:
Har diskutert det med Andre og i community kan vi ha egen flow/valideringsregel som skal løse det (enten ved å informere bruker at maks antall deltakere er nådd eller/og blokkere muligheten for å registrere nye deltakere via community). Men det er best å deploy endringer i preprod/prod og eveutelt teste i preprod hvordan Process builder, valideringsregel funker sammen med community sin flow. Hva synes du om det?